app/mci/pkg/gcp/healthcheck/fake_healthchecksyncer.go

```diff
@@ -15,12 +15,12 @@
 package healthcheck
 
 import (
-	sp "github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/mci/pkg/serviceport"
+	ingressbs "k8s.io/ingress-gce/pkg/backends"
```

Comment:
ingress b.s. ? ;-) how about ingressbes ? or ingbesvc ...

Reply:
Updated to ingressbe